src/metadata.cpp

```diff
@@ -1293,10 +1333,12 @@ void Metadata::InternalData::update_tables(int version, const VersionNumber& cas
       LOG_ERROR("Unable to get column value for 'keyspace_name' or 'columnfamily_name'");
       continue;
     }
+
     if (keyspace_name != temp_keyspace_name) {
       keyspace_name = temp_keyspace_name;
+      keyspace = get_or_create_keyspace(keyspace_name);
     }
-    KeyspaceMetadata* keyspace = get_or_create_keyspace(keyspace_name);
+
     keyspace->add_table(TableMetadata::Ptr(new TableMetadata(columnfamily_name, version, buffer, row)));
```

Comment:
Recommend reverting this some compilers may treat this as error due to `-Werror` (use of an initialized variable keyspace) or ensure keyspace is initialized to NULL.